arrow-core-data/src/main/kotlin/arrow/typeclasses/suspended/BindSyntax.kt

```diff
@@ -23,4 +24,9 @@ interface BindSyntax<F> {
 
   suspend operator fun <A> Kind<F, A>.not(): A =
     bind()
+
+  suspend fun <A> Kind<F, A>.getValue(thisRef: Any?, property: KProperty<*>): A =
+    bind()
 }
+
+operator fun String.getValue(thisRef: Any?, property: KProperty<*>): Char = 'c'
```

Comment:
Why where these  2 `getValue` operators added?

Reply:
I believe the above for the bind syntax to see if we can enable it already to start using it but the String.getValue makes no sense

Reply:
Simon, I think they come from `0.11.0/bio` branch: https://github.com/arrow-kt/arrow-core/pull/74/commits/99bceecd15976cc390ab0eaeb91ee95ad0853bc7